holidayextras/ui-toolkit
# Weather https

#### What does this PR do? (please provide any background)
- We realised our weather fonts and other images could not be served up from HTTPS
- This creates a variable so people can change the path of their assets
- Moves existing weather assets to use this new variable.

#### What tests does this PR have?
None

#### How can this be tested?
- Run `grunt docs`
- Scroll down to the weather component
- Ensure the weather icons are visible and that the fonts are loaded from the same location as in the asset variable

#### Screenshots / Screencast
<img width="856" alt="screen shot 2015-09-30 at 11 36 04" src="https://cloud.githubusercontent.com/assets/778942/10190854/7a2eae3e-6767-11e5-89fb-2b5c8c907331.png">


#### What gif best describes how you feel about this work?
![Batman](https://media.giphy.com/media/6gDSyjaOPwZ4A/giphy.gif)

---
#### Developer Definition of Done/Quality Checklist (for PR author to complete BEFORE code review):

#### Software Engineer or Developer review:

#### Software Engineer or project guru review:
